### DISCUSSION TITLE
本地测试没有问题，但是部署后却显示白屏啊

### DISCUSSION THREAD

**Discussion by Cslj1024** [hexojs/hexo#5350] [Question / Answer]
本地测试没有问题，但是部署后却显示白屏啊

**Comment by liuzz10** [comment]
同样的问题，delopy一切正常，之前网页还是可以access的，突然就白屏了，304 error
![Uploading Screen Shot 2023-11-10 at 9.32.07 AM.png…]()

**Reply by Cslj1024** [reply]
对的,我也没办法了，只能用git进行回滚

**Reply by Cslj1024** [reply]
写博客的话是GitHubPage还是Vercel，那一种好一点？

**Comment by uiolee** [comment]
Can't not reproduce. Please provide more informations. Does it correct in `hexo s`?

And try to use the `Inspector`, `Console` and `Network` tab of browser developer tools to check if the pages are correct.

**Comment by hscspring** [comment]
node 版本问题，你看public下index.html应该是空白的。
降到node12

**Reply by alphaleadership** [reply]
please update nodejs

**Reply by stevenjoezhang** [reply]
Hexo 和最新版 Node.js 的兼容性不存在问题。Node.js 12 在 2022 年就已经处于 End-of-Life 状态，不建议降级使用。

**Reply by alphaleadership** [reply]
go to node 22 and english charset

**Comment by jiehua1995** [comment]
A very possible reason is the setting of url. Check if it was set correctly.
